ecc/bls12-377/hash_to_g1_test.go

```diff
@@ -30,7 +32,7 @@ func TestG1SqrtRatio(t *testing.T) {
 		func(u fp.Element, v fp.Element) bool {
 
 			var seen fp.Element
-			qr := g1SqrtRatio(&seen, &u, &v) == 0
+			qr := hash_to_curve.G1SqrtRatio(&seen, &u, &v) == 0
```

Comment:
This whole test should be in the new `hash_to_curve` package.

Reply:
I'll move this particular test. I could in theory also move rest of the tests (for map and hash methods), by defining the test file to be in `{{ curve }}_test` package, but it would break code coverage tools as they only test the coverage within the same package.

Reply:
Moved